protobuf/README.md

```diff
@@ -11,6 +11,7 @@ Check out the [Flow Access API specification](/docs/content/access-api.md).
 You can use [buf](https://github.com/bufbuild/buf) to generate gRPC client stubs in a variety of languages.
 Please make sure you have `protoc-gen-go-grpc` installed, for example using command
 ```shell script
+go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.28
```

Comment:
I'm not opposed to making this change, but I think we should do it as a separate PR

Reply:
I had added that line for what I used since I couldn't find what version was being used, but just found via https://github.com/onflow/flow/pull/905#issuecomment-1097221140 that it seems to be set in the flow-go makefile as [v1.3.2 ](https://github.com/onflow/flow-go/blob/master/Makefile#L84)

Reply:
ah ok. thanks for fixing the docs!